README.md

```diff
@@ -1 +1,15 @@
-# hw01
\ No newline at end of file
+# hw01
```

Comment:
Overall solid job of meeting all required steps. 

Basic improvements to consider, demonstrate a mastery of understanding proper coding steps by including more material and repeating more of the requested steps. It seems you said you had a good handle on this, which is great, and adding more to it to demonstrate this mastery is good practice. 

Suggestion to include more links than just your email per instructions. For example, including links to some of your favorite websites, or books, or pictures of things or places you like.

Depth of biography could have been expanded with more robust details given the instructions of the assignment; which alluded to a more meaty bio. Your likes, dislikes, places you've traveled to, movies you like, favorite foods, etc.  

Overall solid job for first assignment.